hammers/scripts/lease_stack_reaper.py

```diff
@@ -41,21 +42,19 @@ def __init__(self, user_id, name, email):
         self.user_id = user_id
         self.name = name
         self.email = email
-        self.nodes = {}
+        self.nodes = collections.defaultdict(list)
```

Comment:
I think you've used this before but I keep forgetting it exists - neat!

Reply:
But, the default value was a `set` before - any difference in making it a `list`? Presumably it can't have dupes, but maybe good to keep it a `set` for safety.

Reply:
I changed the code to add the full lease dictionary we get from blazar rather than a tuple. dictionaries can't be added to a set.